config/install/field.storage.node.field_starship_c.yml

```diff
@@ -0,0 +1,20 @@
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+id: node.field_starship_c
+field_name: field_starship_c
```

Comment:
This field machine name doesn't match the content model.
![starships_field_starship_class_machine_name](https://github.com/notS0Solo/swapimigration/assets/1434979/5bdf9802-aff0-43f8-b81b-741cd0ecbe96)

Reply:
Fixed